tests/integration/scrapers/scrapers-new-test.js

```diff
@@ -34,6 +34,7 @@ if (files) {
           const hasErrors = schema.schemaHasErrors(scraper.default, schema.schemas.scraperSchema);
           t.notOk(hasErrors, 'Scraper had no errors');
         } catch (err) {
+          console.log(err);
```

Comment:
I noticed that on test failures the full stack trace of the error wasn't visible. I thought having it visible at this point in the code would be a good addition, since it definitely helped me in debugging.